recipes-ni/ni-systemimage/ni-systemimage.bb

```
@@ -1,17 +1,22 @@
-SUMMARY = "A system imaging utility for NI LinuxRT safemodes"
+SUMMARY = "A system imaging utility for NI NILRT safemodes"
```

Same fixup as 03352049f248463435cdd22e0166e55b0f9b4f4a.

```suggestion
SUMMARY = "A system imaging utility for NILRT safemodes"
```